### ISSUE TITLE
Webpack resolver doesn't catch exports for all of entries

### ISSUE THREAD

**Issue by lancedikson** [import-js/eslint-plugin-import#605]
I settled the plugin and a webpack resolver for my webpack-based project.

```
"eslint-import-resolver-webpack": "^0.6.0",
"eslint-plugin-import": "^1.12.0",
```

Here is my webpack config:

``` js
// ...
var config = {
//...
entry: {
    'widget': [
      path.join(__dirname, 'src', 'js', 'index.js')
    ]
  },
output: {
    filename: '[name].js',
    path: path.join(__dirname, 'lib')
  },
//...

externals: {
    widget: 'VideolyWidget'
  },
//...

/* add extensions to entries */
var excludedNames = ['base'];
fs
  .readdirSync(path.join(__dirname, "src", "extensions"))
  .forEach(function (extension) {
    var extPath = path.join(__dirname, 'src', 'extensions', extension, 'js', 'index.js');
    if (!excludedNames.includes(extension) && fs.existsSync(extPath)) {
      config.entry['extensions/' + extension] = [extPath];
    }
  });
//...
```

As you can see, I'm trying to make external from the `widget` entry point and use it in extensions. So, when I start linting process it says:

```
~/widget/src/extensions/video-catcher/js/index.js
  2:1  warning  'widget' should be listed in the project's dependencies. Run 'npm i -S widget' to add it  import/no-extraneous-dependencies
```

because of the

``` js
import widget from 'widget';
```

But I'm totally sure that the resolver is trying to resolve it and `findExternal` returns `true` here: https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/webpack/index.js#L95

But resolver works because after I installed it I avoided warnings about aliases.

Any ideas why it doesn't use found external and makes warning about it?

**Comment by benmosher** [comment]
This is more or less a duplicate of #479: no-extraneous-dependencies doesn't have enough metadata (yet) to know that `widget` isn't an actual dependency. Working on it! 😅

**Comment by lancedikson** [comment]
@benmosher, thanks for the answer! Good to hear, that development doesn't stop 👍  You could give me some extra details if you want a participant in this feature. I'd like to help if I can :)
Is there any card or issue with ideas about this feature?

**Comment by soryy708** [comment]
Closed as duplicate of https://github.com/import-js/eslint-plugin-import/issues/479
